_posts/2020-05-15-state-of-presto.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "State of Presto"
+author: Manfred Moser
+excerpt_separator: <!--more-->
```

Comment:
This isn't needed since the default excerpt is the first paragraph

Reply:
I know ... its just more explicit that way .. also we should really move that to the _config.yml .. should I do that in a separate PR?

Reply:
After a bit of testing .. if we declare in _config.yml it overrides the first paragraph rule and make the separator required for these posts.

Reply:
I kinda think we should leave it after the first paragraph and remove from header . so I have done that for now.